sierra_adapter/sierra_item_merger/src/main/scala/uk/ac/wellcome/platform/sierra_item_merger/modules/SierraDynamoConfigModule.scala

```diff
@@ -0,0 +1,15 @@
+package uk.ac.wellcome.platform.sierra_item_merger.modules
+
+import com.google.inject.Provides
+import uk.ac.wellcome.finatra.modules.DynamoConfigModule
+import uk.ac.wellcome.models.aws.DynamoConfig
+import javax.inject.Singleton
+
+object SierraDynamoConfigModule extends DynamoConfigModule {
```

Comment:
This could probably be copied to the common lib as well.

Reply:
Same as above: I agree but we have a lot of deducplication of code to do both on the mergers and on the sierra to dynamo ones. I think we should do that separately